server/src/test/java/org/candlepin/util/ContentOverrideValidatorTest.java

```diff
@@ -73,15 +71,15 @@ public void testValidateCollectionBothInvalid() {
         overrides.add(new ContentOverride("label", "baseurl", "value"));
         overrides.add(new ContentOverride("other label", "name", "other value"));
 
-        boolean gotException = false;
         try {
             validator.validate(overrides);
+            fail("Expected exception \"BadRequestException\" was not thrown.");
         }
         catch (BadRequestException bre) {
-            assertEquals("Not allowed to override values for:" +
-                " name, baseurl", bre.getMessage());
-            gotException = true;
+            assertTrue(bre.getMessage().matches(
+                "^Not allowed to override values for: (?:baseurl, name|name, baseurl)"
+            ));
```

Comment:
Is the intent to care if only 1 of X is bad? Test description says BothInvalid, but this will succeed if only one is bad, will it not?